src/applications/check-in/components/pages/ConfirmablePage/index.jsx

```diff
@@ -17,7 +17,6 @@ const ConfirmablePage = ({
   data = {},
   yesAction = () => {},
   noAction = () => {},
-  withBackButton = false,
```

Comment:
I think we still need this. Not every confirmable page has a back button. I think the instance is the contact info on day of check-in